css-ecommerce-page/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+  <div class="container">
+    <div class="new-arrivals-row">
+      <h2>New Arrivals</h2>
+    </div>
+
+    <!-- Empty Column -->
+    <div></div>
```

Comment:
what is the `empty column` being used for? If it is for formatting purposes, you want to avoid using elements for that purpose and CSS should be used.

Reply:
Yes I used them for adding some blank space on the sides of the main columns, now I've deleted the empty columns and added some margin to the container.
Thanks for the clarification 😊